### PR TITLE
Child Workflows With Strong Parent Reference

### PR DESCRIPTION
**Pull request by coltmcnealy-lh**

### CODE REVIEW

File: gradle.properties
Comment (mijailr) [inline gradle.properties:1]: Is this change intended?

**Review by mijailr — APPROVED** [review]
Looks good to me, would be great to have an example though.

File: gradle.properties
Comment (sauljabin) [inline gradle.properties:1]: revert this please

**Review by sauljabin — CHANGES_REQUESTED** [review]

File: gradle.properties
Comment (coltmcnealy-lh) [reply gradle.properties:1]: Still in progress this PR, let me update it to draft

Comment (coltmcnealy-lh) [reply gradle.properties:1]: It's missing tests and examples so I can't merge it. But I will remove that marker.

Comment (sauljabin) [reply gradle.properties:1]: OK, change it to draft 👍🏽

**Comment by coltmcnealy-lh** [comment]
This is ready for review. Only thing missing is python and go sdk's; I have created a ticket and will implement it this weekend. Let's get this merged to unblock the control plane team.

**Review by sauljabin — APPROVED** [review]

**Pull request merged**
